src/testutil-new-tab.html

```diff
@@ -1,12 +1,17 @@
 <html>
```

Comment:
I missed this file last time :-)

Add `<!doctype html>`

Reply:
Done.

Reply:
I also renamed this file to `src/test-util-new-tab.html` to match the punctuation on `src/test-util.ts`.